vendor/assets/stylesheets/rails_admin/themes/myzivi_theme/theming.scss

```diff
@@ -0,0 +1,9 @@
+@import './theming/layout';
+@import './theming/content';
+@import './theming/headers';
+@import './theming/tables';
+@import './theming/stats';
+@import './theming/progress';
+@import './theming/list';
+@import './theming/buttons';
+@import './theming/forms';
```

Comment:
All those files in `vendor/assets/stylesheets/rails_admin/themes` were copied from a different theme and adapted a bit with myzivi style